homeassistant/const.py

```diff
@@ -694,9 +694,19 @@ class UnitOfPressure(StrEnum):
 PRESSURE_PSI: Final = "psi"
 """Deprecated: please use UnitOfPressure.PSI"""
 
+
 # Sound pressure units
+class UnitOfSoundPressure(StrEnum):
+    """Sound pressure units."""
+
+    DECIBEL = "dB"
+    WEIGHTED_DECIBEL_A = "dB(A)"
```

Comment:
Adjusted based on https://github.com/home-assistant/architecture/discussions/837#discussioncomment-4332813

Reply:
As you said, the current variant `dBa` isn't really valid.
I prefer `dB(A)` over `dBA` because I think it makes it clearer it's added information.